Create get_diskusage.js

botondhegedus opened the pull request:
Update index.js

botondhegedus committed:
Add get_diskusage_moodle with external_api
Add get_diskusage_moodle to index.js

jlabusch commented:
CodeClimate has spotted our blatant copy/paste job. Let's refactor, because we know that the Mahara disk plugin is coming as well.

jlabusch commented:
Additionally, we need to handle sites with multiple applications, and the dash needs to know which applications to check. We might need to work with Maria on her Mahara plugin to get the full picture.

botondcatalyst committed:
Merge remote-tracking branch 'upstream/master'

botondcatalyst committed:
Fix coding when error occurs

botondcatalyst committed:
Add "Disk usage" block to Dashboard
- add diskusage_moodle to default.json.example
- remove storage disk space calls
- fix missing jquery.min.map debug notice
